frontend/src/cards/ui/SviAlert.tsx

```diff
@@ -23,6 +23,11 @@ export const findRating = (svi: number) => {
   return "medium";
 };
 
+export const findVerboseRating = (svi: number) => {
+  const rating = findRating(svi);
+  return `${rating[0].toUpperCase() + rating.slice(1)} vulnerability`;
```

Comment:
nice. i always forget JS doesn't have a built in string method to capitalize just the first letter... seems weird they didn't include it. Even CSS has it lol